connaisseur/flask_application.py

```diff
@@ -153,6 +153,17 @@ async def __validate_image(type_index, image, admission_request):
     type_, index = type_index
     logging_context.update(image=original_image)
 
+    # a resource that is being updated, without any image reference changes,
+    # can be automatically skipped, as image validation happened at some
```

Comment:
That's not necessarily true. Deployment could be from before Connaisseur deployment. Would remove that reasoning and maybe replace with link or number of issue that pursuaded you to implement